art/text_dic3.py

```diff
@@ -12498,3 +12498,681 @@
+fancy82_dic = {'a': 'Λ', 'b': 'ɓ', 'c': '¢', 'd': 'Ɗ', 'e': '£', 'f': 'ƒ', 'g': 'ɢ', 'h': 'ɦ', 'i': 'ĩ', 'j': 'ʝ', 'k': 'Қ', 'l': 'Ł', 'm': 'ɱ', 'n': 'ה', 'o': 'ø', 'p': 'Ṗ', 'q': 'Ҩ', 'r': 'Ŕ', 's': 'Ş', 't': 'Ŧ', 'u': 'Ū', 'v': 'Ɣ', 'w': 'ω', 'x': 'Ж', 'y': '¥', 'z': 'Ẑ', 'A': 'Λ', 'B': 'ɓ', 'C': '¢', 'D': 'Ɗ', 'E': '£', 'F': 'ƒ', 'G': 'ɢ', 'H': 'ɦ', 'I': 'ĩ', 'J': 'ʝ', 'K': 'Қ', 'L': 'Ł', 'M': 'ɱ', 'N': 'ה', 'O': 'ø', 'P': 'Ṗ', 'Q': 'Ҩ', 'R': 'Ŕ', 'S': 'Ş', 'T': 'Ŧ', 'U': 'Ū', 'V': 'Ɣ', 'W': 'ω', 'X': 'Ж', 'Y': '¥', 'Z': 'Ẑ', '!': '!', '"': '"', '#': '#', '$': '$', '%': '%', '&': '&', "'": "'", '(': '(', ')': ')', '*': '*', '+': '+', ',': ',', '-': '-', '.': '.', '/': '/', ':': ':', ';': ';', '<': '<', '=': '=', '>': '>', '?': '?', '@': '@', '[': '[', '\\': '\\', ']': ']', '^': '^', '_': '_', '`': '`', '{': '{', '|': '|', '}': '}', '~': '~', '0': '0', '1': '1', '2': '2', '3': '3', '4': '4', '5': '5', '6': '6', '7': '7', '8': '8', '9': '9', ' ': ' '}
+fancy83_dic = {'a': 'λ', 'b': 'ß', 'c': 'Ȼ', 'd': 'ɖ', 'e': 'ε', 'f': 'ʃ', 'g': 'Ģ', 'h': 'ħ', 'i': 'ί', 'j': 'ĵ', 'k': 'κ', 'l': 'ι', 'm': 'ɱ', 'n': 'ɴ', 'o': 'Θ', 'p': 'ρ', 'q': 'ƣ', 'r': 'ર', 's': 'Ș', 't': 'τ', 'u': 'Ʋ', 'v': 'ν', 'w': 'ώ', 'x': 'Χ', 'y': 'ϓ', 'z': 'Հ', 'A': 'λ', 'B': 'ß', 'C': 'Ȼ', 'D': 'ɖ', 'E': 'ε', 'F': 'ʃ', 'G': 'Ģ', 'H': 'ħ', 'I': 'ί', 'J': 'ĵ', 'K': 'κ', 'L': 'ι', 'M': 'ɱ', 'N': 'ɴ', 'O': 'Θ', 'P': 'ρ', 'Q': 'ƣ', 'R': 'ર', 'S': 'Ș', 'T': 'τ', 'U': 'Ʋ', 'V': 'ν', 'W': 'ώ', 'X': 'Χ', 'Y': 'ϓ', 'Z': 'Հ', '!': '!', '"': '"', '#': '#', '$': '$', '%': '%', '&': '&', "'": "'", '(': '(', ')': ')', '*': '*', '+': '+', ',': ',', '-': '-', '.': '.', '/': '/', ':': ':', ';': ';', '<': '<', '=': '=', '>': '>', '?': '?', '@': '@', '[': '[', '\\': '\\', ']': ']', '^': '^', '_': '_', '`': '`', '{': '{', '|': '|', '}': '}', '~': '~', '0': '0', '1': '1', '2': '2', '3': '3', '4': '4', '5': '5', '6': '6', '7': '7', '8': '8', '9': '9', ' ': ' '}
+fancy84_dic = {'a': 'Λ', 'b': 'Б', 'c': 'Ͼ', 'd': 'Ð', 'e': 'Ξ', 'f': 'Ŧ', 'g': 'g', 'h': 'h', 'i': 'ł', 'j': 'j', 'k': 'К', 'l': 'Ł', 'm': 'm', 'n': 'Л', 'o': 'Ф', 'p': 'p', 'q': 'Ǫ', 'r': 'Я', 's': 's', 't': 't', 'u': 'u', 'v': 'v', 'w': 'Ш', 'x': 'Ж', 'y': 'Џ', 'z': 'z', 'A': 'Λ', 'B': 'Б', 'C': 'Ͼ', 'D': 'Ð', 'E': 'Ξ', 'F': 'Ŧ', 'G': 'G', 'H': 'H', 'I': 'ł', 'J': 'J', 'K': 'К', 'L': 'Ł', 'M': 'M', 'N': 'Л', 'O': 'Ф', 'P': 'P', 'Q': 'Ǫ', 'R': 'Я', 'S': 'S', 'T': 'T', 'U': 'U', 'V': 'V', 'W': 'Ш', 'X': 'Ж', 'Y': 'Џ', 'Z': 'Z', '!': '!', '"': '"', '#': '#', '$': '$', '%': '%', '&': '&', "'": "'", '(': '(', ')': ')', '*': '*', '+': '+', ',': ',', '-': '-', '.': '.', '/': '/', ':': ':', ';': ';', '<': '<', '=': '=', '>': '>', '?': '?', '@': '@', '[': '[', '\\': '\\', ']': ']', '^': '^', '_': '_', '`': '`', '{': '{', '|': '|', '}': '}', '~': '~', '0': '0', '1': '1', '2': '2', '3': '3', '4': '4', '5': '5', '6': '6', '7': '7', '8': '8', '9': '9', ' ': ' '}
+reverse1_dic = {
+    'a': '9',
+    'b': '8',
+    'c': '7',
```

Comment:
It seems `rev1` and `rev2` have some problems, for example `rev1` mapped `c` to `7`!!!!

Reply:
Smiley's reverse fonts map the input :
``
abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ!"#$%&'()*+,-./:;<=>?@[\]^_`{|}~0123456789
``
to :
``
9876543210~}|{`_^]\[@?>=<;:/.-,+*)('&%$#"!zʎxʍʌnʇsɹbdouɯๅʞſıɥɓɟǝpɔqɐzʎxʍʌnʇsɹbdouɯๅʞſıɥɓɟǝpɔqɐ
``

so that first character maped to last one and ...

Reply:
It seems there is no one-to-one correspondence in these fonts.
Please remove them ;-)